### PR TITLE
add some words about keeping Slack and GitHub handles in sync

### PR DESCRIPTION
**Pull request by shiftkey**
Fixes https://github.com/kubernetes/website/issues/5952

### CODE REVIEW

File: contributors/devel/welcome-to-kubernetes-new-developer-guide.md [hunk 1]
```diff
@@ -16,8 +16,10 @@ able to pick up issues, write code to fix them, and get your work reviewed and
 merged.
 
 If you have questions about the development process, feel free to jump into our
-[Slack Channel](http://slack.k8s.io/) or join our [mailing
-list](https://groups.google.com/forum/#!forum/kubernetes-dev).
+[Slack channel](http://slack.k8s.io/) or join our [mailing
+list](https://groups.google.com/forum/#!forum/kubernetes-dev). If you join the
+Slack channel it is recommended to set your Slack display name to your GitHub
```
Comment (cblecker) [inline contributors/devel/welcome-to-kubernetes-new-developer-guide.md]: nit: s/channel/workspace/

**Comment by cblecker** [comment]
Also, please squash your commits :)
/ok-to-test

**Comment by shiftkey** [comment]
Squashed and rebased on `master` 🌕

**Comment by cblecker** [comment]
Fantastic! Thanks for this :)
/lgtm

**Pull request merged**